api/src/models/service_config/mod.rs

```diff
@@ -26,23 +26,26 @@
 use crate::models::service::ContainerType;
 use crate::models::Image;
 pub use environment::{Environment, EnvironmentVariable};
+use secstr::SecUtf8;
 use serde::Deserialize;
 use serde_value::Value;
 use std::collections::BTreeMap;
 use std::hash::Hash;
 use std::path::PathBuf;
-
 mod environment;
 mod templating;
 
-#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq)]
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq)]
```

Comment:
Why did you change that line?

Reply:
Hash is not implemented by SecUtf8. Also removing it did not change behavior. As discussed maintaining the derives without Hash